wp-content/civi-extensions/goonjcustom/api/v3/Goonjcustom/CollectionCampCron.php

```diff
@@ -128,25 +90,23 @@ function civicrm_api3_goonjcustom_collection_camp_cron($params) {
         ->execute();
 
       // Send completion notification.
-      if (!$logisticEmailSent && $recipientId && $endDateFormatted <= $todayFormatted) {
-        // Get recipient email.
-        $email = Email::get(TRUE)
-          ->addWhere('contact_id', '=', $recipientId)
+      if (!$logisticEmailSent && $campAttendedById && $endDateFormatted <= $todayFormatted) {
```

Comment:
Now that we have separated the two crons we can update this query to add the where clause for `camp to be attended by is not null`
```suggestion
      if (!$logisticEmailSent && $endDateFormatted <= $todayFormatted) {
```

Reply:
https://github.com/ColoredCow/goonj/commit/171aca0714e10a15174d5a11ec26440cf75cb6f0#diff-6c9321ddca11b05d6d58cc4d5ed8382d2e22108f89d7519b7559e4c33ad5f58eL61